test/func/mocha.opts

```diff
@@ -0,0 +1,3 @@
+test/func/spec/**/*.spec.js
+--recursive
+--timeout 50000
```

Comment:
Removed timeout in lieu of https://github.com/FormidableLabs/builder-docs-archetype/compare/feat-triggerBundle?expand=1